test/upgrade-e2e/post_upgrade_test.go

```diff
@@ -101,16 +101,47 @@ func TestClusterExtensionAfterOLMUpgrade(t *testing.T) {
 	require.EventuallyWithT(t, func(ct *assert.CollectT) {
 		assert.NoError(ct, c.Get(ctx, types.NamespacedName{Name: testClusterExtensionName}, &clusterExtension))
 		cond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1.TypeInstalled)
-		if !assert.NotNil(ct, cond) {
```

Comment:
this will never return false - either it returns true or fails

Reply:
Yup, I agree.